### PR TITLE
Added Daily Quote feature

### PR DESCRIPTION
**Pull request by DevoAbhi**
## Types of changes - Feature added.

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x ] My changes generate no new warnings
- [ x ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

canva-quote-desktop

canva-quote-mobile

## Other information

Please mention if any other changes are required in the PR.
Thank You.

### CODE REVIEW

File: src/app/components/dashboard/dashboard.component.scss [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+
+
+
 .container{
   .buttons {
     button {
       margin-right:10px;
     }
   }
+
```
Comment (kingavatar) [inline src/app/components/dashboard/dashboard.component.scss:11]: Don't Duplicate the class.
Add it below line no: 5

File: src/app/components/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -11,7 +12,10 @@ declare var $: any;
 })
 export class DashboardComponent implements OnInit {
 
-  constructor(private route: Router, private apiService: RestService) { }
+  public quote: DailyQuote;
+  constructor(private route: Router, private apiService: RestService) {
+    this.getQuote();
```
Comment (goliakshay357) [inline src/app/components/dashboard/dashboard.component.ts:17]: Please call this function inside NgonInit() of the component.

**Review by goliakshay357 — COMMENTED** [review]
image
Quote has to be in the bottom of the page. 
One silution would be:
1) Make container height to 100vh 
2) Place the quote div and container on the same DOM levels

**Comment by DevoAbhi** [comment]
Kindly review the latest commit.

**Comment by DevoAbhi** [comment]
I have added some dummy quotes so that if the API fails to get a response, the user can still see a quote displayed.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @DevoAbhi I tested this PR and the quote is not appearing as expected. Maybe you should make it stick to the bottom by using something like below for the `quote` div.
```
position: fixed;
bottom: 8px;
```

Also please remove the commented codes if it's not required.

File: src/app/components/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -361,6 +364,16 @@ export class DashboardComponent implements OnInit {
...
+  getQuote() {
+    this.apiService.getDailyQuote()
+    .then((quote: DailyQuote) => {
+      this.quote = quote;
+    })
+    .catch((err) => {
+      // Here we store a random dummyQuote to the quote property.
+      this.quote= this.apiService.getDummyQuote();
+    });
+
```
Comment (Kajol-Kumari) [inline src/app/components/dashboard/dashboard.component.ts:376]: Add the missing paranthesis(`}`) here. This is making your build fail

File: src/app/components/dashboard/dashboard.component.html [hunk 4]
```diff
@@ -24,10 +23,13 @@ <h3 style="color: black;"><strong>Welcome {{Username}}!👋</strong></h3>
     </div>
   </div>
 
+  <!-- <input type="text" [(ngModel)]="folderName"> -->
```
Comment (Kajol-Kumari) [inline src/app/components/dashboard/dashboard.component.html:26]: If this comment line isn't required, please remove it.

File: src/app/components/dashboard/dashboard.component.scss [hunk 5]
```diff
@@ -57,15 +63,15 @@
 
   }
 
-
   @media (max-width:1200px) {
     .cb-files-container {
       columns: 3;
       width: calc(100%-40px);
-      cb-box-sizing: border-cb-box;
+      // cb-box-sizing: border-cb-box;
```
Comment (Kajol-Kumari) [inline src/app/components/dashboard/dashboard.component.scss:70]: please remove the commented line

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @DevoAbhi The feature is working fine. Justr address the minor requested changes and it will be good to go!

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for incorporating all the request changes @DevoAbhi
LGTM 👍

**Comment by DevoAbhi** [comment]
Thank you for merging my PR.
Thank you @Kajol-Kumari , @goliakshay357 and @kingavatar for reviewing my PR.

**Pull request merged**